core/domain/rights_manager.py

```diff
@@ -618,16 +991,19 @@ def _release_ownership_of_activity(committer_id, activity_id, activity_type):
 
 def _change_activity_status(
         committer_id, activity_id, activity_type, new_status, commit_message):
-    """Change the status of an activity. Commits changes.
+    """Changes the status of an activity.
 
     Args:
-    - committer_id: str. The id of the user who is performing the update
-        action.
-    - activity_id: str. The id of the collection or activity.
-    - activity_type: str. One of feconf.ACTIVITY_TYPE_EXPLORATION or
-        feconf.ACTIVITY_TYPE_COLLECTION.
-    - new_status: str. The new status of the exploration.
-    - commit_message: str. The human-written commit message for this change.
+        committer_id: str. ID of user who is performing the update action.
+        activity_id: str. ID of collection or activity.
+        activity_type: The type of activity. Either one of these:
+            feconf.ACTIVITY_TYPE_EXPLORATION
+            feconf.ACTIVITY_TYPE_COLLECTION
+        new_status: str. The new status of exploration.
```

Comment:
I think this should be written as "The new status of the activity." instead of exploration. Done it accordingly.